packages/neos-ui/src/Containers/EditModePanel/style.css

```diff
@@ -38,8 +38,8 @@
     margin-right: 1rem;
     white-space: nowrap;
 
-    &:last-child {
-        margin-right: 0;
```

Comment:
Maybe we should just remove that property and live with the fact that the last item has also a 1rem margin-right - this would reduce complexity, any thoughts on this?

Reply:
Would be also a valid solution. I don't care if the last item has the margin or not ;)